quickwit/quickwit-cli/tests/cli.rs

```diff
@@ -142,23 +142,28 @@ async fn test_cmd_create() -> Result<()> {
     Ok(())
 }
 
-#[test]
-fn test_cmd_create_on_existing_index() -> Result<()> {
+#[tokio::test]
+async fn test_cmd_create_on_existing_index() -> Result<()> {
```

Comment:
```suggestion
async fn test_cmd_create_on_existing_index() {
```

Reply:
You get a better error message when a test fails on `foo().unwrap()` rather than `foo?`.